core/src/main/java/ai/timefold/solver/core/impl/domain/solution/descriptor/DefaultPlanningVariableDiff.java

```diff
@@ -27,7 +28,10 @@ public String toString() {
                   Old value: %s
                   New value: %s
                 """
-                .formatted(variableMetaModel.isGenuine() ? "genuine" : "shadow", variableMetaModel.name(),
+                .formatted(
```

Comment:
Should we add a base method to return the description and override it in child classes?

Reply:
The method would have to exist on the public API. This is not something that a user needs, therefore I do not clutter the public API with this.